packages/components/managed-data-table/src/data-table-settings/droppable-panel/draggable-tag/draggable-tag.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
```

Comment:
Is this a duplicate of https://github.com/commercetools/ui-kit/pull/1416/files?file-filters%5B%5D=.js&file-filters%5B%5D=.lock&file-filters%5B%5D=.md#diff-402776c3269cdf1e37c07c20da4d895a ?

Reply:
Nice catch! I don't know how but it looks like I've moved all the tag files to a folder and the original ones remained there 🤦